python/lsst/pipe/base/argumentParser.py

```diff
@@ -1279,6 +1279,10 @@ def __call__(self, parser, namespace, values, option_string):
                 namespace.log.setLevel(logLevel)
             else:
                 lsstLog.Log.getLogger(component).setLevel(logLevel)
+            # Python logging levels are same as lsst.log divided by 1000,
+            # logging does not have TRACE level by default but it is OK to use
+            # that numeric level and we may even add TRACE later.
+            logging.getLogger(component).setLevel(logLevel//1000)
```

Comment:
We have `LogHandler.translateLevel()` to multiply by 1000; should we add a routine to divide by 1000 as well, since it appears here as well as in https://github.com/lsst/log/blob/master/python/lsst/log/log/logContinued.py#L123-L124 ?

Reply:
Yes, I though about that while reviewing Tim's ticket 🙂 I think I'll do it now, need to think where these two methods should be (either a module level method or in `Log` class or `LogHandler`)